src/main/controllers/Logon.controller.ts

```diff
@@ -17,7 +18,7 @@ import {jsonToCsv} from '../util/CsvHandler';
 export class LogonController {
   private logger: LoggerInstance = Logger.getLogger('LogonsController');
 
-  private service = new LogonService();
+  private service = new LogonService(new AuthService());
```

Comment:
PR looks fine. Passing in new AuthService() necessary though as you are instantiating it anyway in the base class?